scripts/install.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+set -e
+
+DOWNLOAD_URL="https://github.com/redhat-developer/ocdev/releases/download/v0.0.1"
+OCDEV_VERSION="latest"
+BINARY_NAME="ocdev"
+CANCELLATION_TIMEOUT=15
+INSTALLATION_PATH="/usr/local/bin/"
+
+SUPPORTED_DISTRIBUTIONS="
+darwin-amd64
+linux-amd64
+linux-arm
+windows-amd64
+"
+
+echo_stderr ()
+{
+    echo "$@" >&2
+}
+
+command_exists() {
+	command -v "$@" > /dev/null 2>&1
+}
+
+check_platform() {
+    kernel="$(uname --kernel-name)"
+    if [ "$(uname --machine)" = "x86_64" ]; then
+        arch="amd64"
+    fi
+
+    platform_type="${kernel,,}-$arch"
+
+    if ! echo "$SUPPORTED_DISTRIBUTIONS" | grep "$platform_type" > /dev/null; then
+        echo_stderr "
+The installer has detected your platform to be $platform_type, which is
+currently not supported by this installer script.
+
+Please visit the following URL for detailed installation steps:
+https://github.com/redhat-developer/ocdev/#installation
+
+        "
+        exit 1
+    fi
+    echo "$platform_type"
+}
+
+install_ocdev() {
+    echo "Starting ocdev installation..."
+    echo
+    echo "Detecting distribution..."
+    echo
+
+    platform="$(check_platform)"
+    echo "Detected platform: $platform"
+    echo
+
+    BINARY_URL="$DOWNLOAD_URL/$BINARY_NAME-$platform.gz"
+
+    if command_exists ocdev; then
+        echo "
+ocdev version \"$(ocdev version)\"is already installed on your system, running this installer script might case issues with your current installation.
+Press Ctrl+C to abort installation.
+"
+        ( set -x; sleep $CANCELLATION_TIMEOUT )
+    fi
+
+    echo "Downloading ocdev from $BINARY_URL"
+    curl -Lo ocdev.gz $BINARY_URL
+    echo "Extracting ocdev.gz"
+    gunzip -d ocdev.gz
+    echo "Setting execute permissions on ocdev"
+    chmod +x ocdev
+    echo "Moving ocdev binary to $INSTALLATION_PATH"
+    sudo mv ocdev "$INSTALLATION_PATH"
```

Comment:
sudo, doesn't have to be installed on some system, this script should check if it is present it at the beginning and error

Reply:
done, added logic to check for root, if not, check for sudo, if not check for su, if not fail